### DISCUSSION TITLE
Bypass papers antixray if server texturepack accepted

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#671] [__Dismissed__] [answered]
papers antixray makes some players fps drop more then usual it would be cool to have a option that if players accept the server ressourcepack that the paper antixray will be disabled for them so u could use a antixray texturepack for those who have low fps so they dont drop more and then the paper antixray for players who doesent need to care about theyr fps

**Comment by sph432** [comment]
Well, it's a great idea as that's a real problem.
Sadly it's also a horrible idea as one can mod a client to accept and download the respack and then ignore it :disappointed:

**Comment by katekerllenevich** [comment, marked as answer]
The only reason FPS drops is if you add `air` to the list of replacement blocks.
